vendor/github.com/containers/image/types/types.go

```diff
@@ -6,7 +6,6 @@ import (
 	"time"
```

Comment:
This file should have been removed; this suggests that some non-v4/-prefixed references remain.

Reply:
Yes. I missed some. Did it manually - `gomove` did the trick now.